apps/multisig/src/domains/multisig/vaults-scanner.ts

```diff
@@ -79,7 +79,9 @@ export const vaultsOfAccount = selector({
     data.accountExtrinsics.forEach(tx => {
       try {
         // get the signer that signed the transaction
-        const signer = Address.fromPubKey(parseCallAddressArg(tx.extrinsic.signer))
+        const signer =
+          Address.fromPubKey(parseCallAddressArg(tx.extrinsic.signer)) ||
```

Comment:
Since this pattern of `Address.fromPubKey(address) || Address.fromSs58(address)`, or `isEthereumAccount ? Address.fromSs58(address) : Address.fromPubKey(address)` is frequently used, it might be beneficial to build it into the `Address` class as a method.